go/api/commands.go

```diff
@@ -0,0 +1,52 @@
+// Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0
+
+package api
+
+// StringCommands defines an interface for the "String Commands" group of Redis commands for standalone and cluster clients.
+//
+// See [redis.io] for details.
+//
+// [redis.io]: https://redis.io/commands/?group=string
+type StringCommands interface {
+	// Set the given key with the given value. The return value is a response from Redis containing the string "OK".
+	//
+	// See [redis.io] for details.
+	//
+	// For example:
```

Comment:
```suggestion
	// Example:
```
to follow other clients